### PR TITLE
[Table] Fix row border for ghost cells

### LINKED ISSUE #972: Ghost and loading cells styling doesn't divider between row headers and row contents
Seen on Chrome & Table 1.10
With row headers, but without any actual rows, ghost cells don't show the divider between the row headers and the row contents

image

Related context: https://pl.ntr/eI

### PR DESCRIPTION
**Pull request by tgreenwatts**
#### Fixes #972 

NOTE: Left the example with no rows temporarily, before merging I will put it back! 

Pretty much making this up, so please tell me if I'm doing it wrong

### CODE REVIEW

File: packages/table/preview/index.tsx [hunk 1]
```diff
@@ -250,7 +250,7 @@ ReactDOM.render(
 );
 
 ReactDOM.render(
-    getTableComponent(2, 2, {}, {
+    getTableComponent(0, 0, {}, {
```
Comment (tgreenwatts) [inline packages/table/preview/index.tsx]: I will put this back before merging, left for testing

**Comment by llorca** [comment]
Right idea! We gotta make sure we test all situations this time. I only see a couple of issues so far, one with the last row headers:
image

One with an empty ghost table (Safari only, but it wasn't broken before so it's a regression):
image

**Comment by tgreenwatts** [comment]
Updated! I think it works this time, @cmslewis and I could not repro the Safari issue

**Comment by tgreenwatts** [comment]
Reset the example so you can see with both a small table with ghost cells & an empty table with ghost cells

**Comment by cmslewis** [comment]
A lot of our tables don't render on our preview page, because the maximum stack size is exceeded. But the few tables that did appear looked good.

**Comment by tgreenwatts** [comment]
@llorca can I get a review here?

**Pull request merged**